src/components/WalletModal/index.tsx

```diff
@@ -182,13 +200,18 @@ export default function WalletModal({
     }
 
     connector &&
-      activate(connector, undefined, true).catch((error) => {
-        if (error instanceof UnsupportedChainIdError) {
-          activate(connector) // a little janky...can't use setError because the connector isn't set
-        } else {
-          setPendingError(true)
-        }
-      })
+      activate(connector, undefined, true)
+        .then(async () => {
+          const walletAddress = await connector.getAccount()
+          logMonitoringEvent({ walletAddress })
```

Comment:
Should we add this?  https://github.com/gnosis/cowswap/blob/e4ad97cdba5b3b3f1481fdf38f1e9f32f5e0be11/src/custom/components/WalletModal/WalletModalMod.tsx#L214

we should not leak info to them, but maybwe we want to make use of some of these log/monitoring features

Reply:
Since for this we have Mod file this component , this isn't used but I will add this code to mod and comment it out for now.